fheng/environment-proxy
# PoC for environment proxy

Verification 
#
- Create environment through the studio
- Go to that environment project and follow this README
- Deploy an app into the environment
- Take a look at `/var/lib/haproxy/conf/os_edge_be_http.map` and get the path of the service
- Visit the auto generated router url and append the path and make sure the app is reachable :pray:
- If not :sob:

@maleck13 @wtrocki
